extensions/amp-auto-ads/0.1/ad-network-config.js

```diff
@@ -97,6 +97,7 @@ class AdSenseNetworkConfig {
     return buildUrl('//pagead2.googlesyndication.com/getconfig/ama', {
       'client': this.autoAmpAdsElement_.getAttribute('data-ad-client'),
       'plah': canonicalHostname,
+      'url': docInfo.canonicalUrl,
       'ama_t': 'amp',
```

Comment:
Do you limit the length of the url?  We do so for A4A ad requests (4k I believe).  Consider doing so and putting url last so that it gets truncated (don't lose ama_t)